app/src/interfaces/select-color/select-color.vue

```diff
@@ -369,8 +369,9 @@ function useColor() {
 
 	position: relative;
 	box-sizing: border-box;
-	width: calc(var(--input-height) - 12px);
-	max-height: calc(var(--input-height) - 12px);
+	margin-left: -8px;
+	width: calc(var(--input-height) - 20px);
+	max-height: calc(var(--input-height) - 20px);
```

Comment:
The `margin-left` is referred from File interface, and `width` & `max-height` here calculates to `40px`, similar to File interface's width and height:

https://github.com/directus/directus/blob/b481e253317d9f84430a22c5a95551f6fe448fc3/app/src/interfaces/file/file.vue#L297-L299